src/lib/libast/tests/run_test.sh

```diff
@@ -6,22 +6,20 @@
 #
 # Make sure when called via `meson test` we've got the expected args.
 #
-function log_info {
-    typeset lineno=$1
-    print -r "<I> run_test[$lineno]: ${@:2}"
+log_info() {
+    echo "<I> run_test[$1]: $2"
```

Comment:
FWIW this isn't equivalent to the original version which allows the message to be passed as multiple words. This version requires the message to be quoted in order to be passed as a single arg. In practice this probably doesn't matter because we should probably always be quoting the message so it is a single word. My thought was that if someone did write

```
log_info this is the message
```

rather than

```
log_info "this is the message"
```

it should still display the entire message.